sdk/mosaic.go

```diff
@@ -17,52 +17,79 @@ type MosaicService service
 // mosaics get mosaics Info
 // @get /mosaic/{mosaicId}
 func (ref *MosaicService) GetMosaic(ctx context.Context, mosaicId *MosaicId) (mscInfo *MosaicInfo, resp *http.Response, err error) {
-
 	mscInfoDTO := &mosaicInfoDTO{}
 	resp, err = ref.client.DoNewRequest(ctx, "GET", pathMosaic+mosaicId.toHexString(), nil, mscInfoDTO)
 
 	if err != nil {
 		return nil, resp, err
 	}
 
-	mscInfo, err = mscInfoDTO.setMosaicInfo()
+	mContext, err := prepareMosaicContext(ref.client)
```

Comment:
First - do request, second - check client?
We need make all check before other diels